src/resources/views/volumesSidebar.blade.php

```diff
@@ -1,3 +1,12 @@
 @if (($user->can('edit-in', $volume) || $user->can('sudo')) && $volume->isImageVolume() && !$volume->hasTiledImages())
```

Comment:
Global admins actually can't create annotations in projects they don't belong to (and the `edit-in` authorization check will fail), so the tab can be hidden for them, too.

```suggestion
@if ($user->can('edit-in', $volume) && $volume->isImageVolume() && !$volume->hasTiledImages())
```